.github/workflows/build-test-package.yml

```diff
@@ -310,8 +333,11 @@ jobs:
               ../VTK
         ninja
         EOF
-        chmod +x ./git-configure-build-vtk.sh
-        /tmp/dockcross-manylinux-x64 -a "-v $PWD/ITKPythonPackage:/ITKPythonPackage" ./git-configure-build-vtk.sh
+        chmod u+x ./git-configure-build-vtk.sh
+
+        source "./ITKPythonPackage/scripts/dockcross-manylinux-set-vars.sh"
+        docker run --privileged --rm tonistiigi/binfmt --install all
+        docker run -v $(pwd):/work/ --rm ${CONTAINER_SOURCE} /work/git-configure-build-vtk.sh
```

Comment:
It looks like `git-configure-build-vtk.sh` also needs to be updated to `pip install ninja` and any other dependencies.

Reply:
I'm curious about why we don't seem to be getting messages from `pip install` indicating that Ninja was installed.

We manually build Ninja in ITKPythonPackage. Perhaps that is necessary here as well?

https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/internal/manylinux-build-common.sh#L71